### ISSUE TITLE
word error rate

### ISSUE THREAD

**Issue by kubumiro** [hgstudent/las#3]
Have you tried to train the model on e.g. Librispeech dataset? I would like to see the word error rate.

**Comment by ServerSideHannes** [comment]
I have yet to train the model on any large dataset. I am planning on doing so when time allows me and will of course post the results here.

**Comment by kubumiro** [comment]
Ok thanks, btw which loss is commonly used for this task? Is categorical crossentropy enough?

**Comment by ServerSideHannes** [comment]
Yes. You can apply cross entropy over a one hot encoded vector where each entry represent a token :)
